thrifty-schema/src/main/java/com/microsoft/thrifty/schema/Constant.java

```diff
@@ -29,11 +29,20 @@
 
 public class Constant extends Named {
     private final ConstElement element;
+    private final Map<NamespaceScope, String> namespaces;
```

Comment:
It looks like this is duplicating the base class field `Named#namespaces` - what's the rationale for that?  Do we need to be able to assign to the field from here, or can we get by with accessing it from the `namespaces()` method?  If not, let's consider changing the original field from `private` to `protected`.